### PR TITLE
Cannot create node source from RM web portal

### LINKED ISSUE #248: Cannot create node source from RM web portal
Codé has discovered this issue recently.

When you try to create a node with _LocalInfrastructure_ and _StaticPolicy_ from the Resource Manager interface, the operation does not complete and a new tab is opened.

Below are screenshots that show the issue:

capture d ecran de 2015-10-30 17-20-50
capture d ecran de 2015-10-30 17-20-52

### PR DESCRIPTION
**Pull request by lpellegr**
This PR closes #248.

### CODE REVIEW

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Jenkins build is not passing because PR plugin [seems to have troubles with my PR description](http://jenkins.activeeon.com/job/scheduling-portal-pull-requests/23/console). Consequently, I have triggered the build manually. [All tests are passing](http://jenkins.activeeon.com/job/scheduling-portal-pull-requests/24/). 

@ow2-proactive/developers Could two people please review the changes?

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NSCreationWindow.java [hunk 1]
```diff
@@ -208,8 +206,9 @@ public void onFormItemClick(FormItemIconClickEvent event) {
                             .substring(inf.getPluginName().lastIndexOf('.') + 1);
                     values.put(inf.getPluginName(), shortName);
 
-                    ArrayList<FormItem> forms = new ArrayList<FormItem>();
-                    for (Field f : inf.getConfigurableFields()) {
+                    List<Field> configurableFields = inf.getConfigurableFields();
```
Comment (marcocast) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NSCreationWindow.java]: This logic seems the same of the previous one. Could we extract some common method here?

**Comment by cdiop** [comment]
OK for me

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NSCreationWindow.java (hunk 1 above)
Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NSCreationWindow.java]: @marcocast Done.

**Comment by marcocast** [comment]
:+1:

**Pull request merged**